lms/envs/common.py

```diff
@@ -590,6 +590,18 @@
     }
 }
 
+COURSE_MODE_DEFAULTS = {
+    'bulk_sku': None,
+    'currency': 'usd',
+    'description': None,
+    'expiration_datetime': None,
+    'min_price': 0,
+    'name': 'Audit',
+    'sku': None,
+    'slug': 'audit',
```

Comment:
I'm a little concerned about this being a string instead of referencing the `AUDIT` var.  Any way we could do this so that we use a named variable that is referenced both here and in the models.  The only way I can think of doing that is by pulling those values to common as well but that doesn't seem very clean either.